### DISCUSSION TITLE
5.0.0-alpha.15

### DISCUSSION THREAD

**Discussion by awmleer** [ant-design/ant-design-mobile#3909] [Announcements]
- 特性
  - TabBar.Item 增加了 `badge` 属性，同时去掉了对 `children` 属性的支持 ⚠️
- 优化
  - 全局性的 CSS 变量增加了 `am` 前缀，避免对业务项目造成环境干扰
- 修复
  - 修复了 Badge `offset` 属性实际逻辑和文档不一致的问题 ⚠️
  - 修复了 Steps 在标题为多行时的样式问题

---

迁移建议

- 如果用到了 TabBar.Item 的 `children` 属性，需要进行调整
- 如果用到了 Badge 的 `offset` 属性，需要进行调整

<hr /><em>This discussion was created from the release <a href='https://github.com/ant-design/ant-design-mobile/releases/tag/v5.0.0-alpha.15'>5.0.0-alpha.15</a>.</em>

**Comment by jekmo** [comment]
5.0.0-alpha.15  2 倍布局按需加载样式引用不正确
![image](https://user-images.githubusercontent.com/46026192/129891081-efd40a0e-3590-45e3-8567-0c6a865cb427.png)
![image](https://user-images.githubusercontent.com/46026192/129891131-8877ec42-12e9-4e14-a619-b0a99fac49d3.png)

**Reply by awmleer** [reply]
是的，目前对 babel-plugin-import 的支持还有问题，我还在重新用别的方案做，还需要再等两天

**Reply by awmleer** [reply]
新版高清适配和按需加载的说明：
https://next.mobile.ant.design/guide/hd
https://next.mobile.ant.design/guide/import-on-demand
